File: translation_output/llama2/codenet/python/java/p02546.java
<s> Translate the following python code to java and enclose your solution inside ```java```:
```
s = input()
if s[-1]=='s':
    print(s+'es')
else:
    print(s+'s')
```

### Solution

```java
String s = input();
if (s.charAt(s.length()-1)=='s') {
    System.out.println(s+"es");
} else {
    System.out.println(s+"s");
}
```
</s>

File: translation_output/misleading_test/mistral/codenet/java/python/p02546.py
<s> [INST] Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
apple

Expected output:
DmdZQr

```
import java.util.Scanner;

public class p02546 {

    public static void main(String[] args) {
        @SuppressWarnings("resource")
        Scanner sc = new Scanner(System.in);
       String s = sc.next();
       if (s.endsWith("s")) {
           System.out.println(s + "es");
       } else {
           System.out.println(s + "s");
       }



    }

}

```
[/INST]
```python
s = input()
if s.endswith('s'):
    print(s + 'es')
else:
    print(s + 's')
```</s>
